File: app/src/main/java/com/philsoft/metrotripper/activity/BaseActivity.java
package com.philsoft.metrotripper.activity;

import android.app.Activity;
import android.os.Bundle;

import com.android.volley.RequestQueue;
import com.philsoft.metrotripper.app.MetroTripperApplication;

/**
 * Created by polson on 10/15/14.
 */
public class BaseActivity extends Activity {

    protected RequestQueue requestQueue;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        requestQueue = ((MetroTripperApplication) getApplication()).getRequestQueue();
    }
}

File: app/src/main/java/com/philsoft/metrotripper/activity/BaseActivity.kt
package com.philsoft.metrotripper.activity

import android.annotation.SuppressLint
import android.support.v4.app.FragmentActivity

@SuppressLint("Registered")
open class BaseActivity : FragmentActivity() {

    var isPaused = false;

    override fun onPause() {
        isPaused = true
        super.onPause()
    }

    override fun onResume() {
        super.onResume()
        isPaused = false
    }
}
